.github/workflows/build.yml

```diff
@@ -1,6 +1,12 @@
 name: Build
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - "*"
```

Comment:
nitpick: You don't need this afaik.

Reply:
so an empty key `pull_request` is avalid you think ?
I can try...

Reply:
uh, you were right